v2/tweet_stream.go

```diff
@@ -23,14 +27,54 @@ const (
 	// ErrorMessageType is the error system message type
 	ErrorMessageType SystemMessageType = "error"
 
-	tweetStart = "data"
+	tweetStart  = "data"
+	keepAliveTO = 11 * time.Second
```

Comment:
I'm not totally clear what this is set to 11 seconds for...let me know if I'm missing something obvious though.

Reply:
I'm assuming you are allotting time to wait for a heartbeat at 10 seconds as described in their docs...just confirming at this point.

Reply:
Yes, it was based off of the documentation of a 10 second heartbeat and I was giving a second buffer on that.